### ISSUE TITLE
volterra_service_policy - panic: interface conversion: interface {} is *schema.Set, not bool

### ISSUE THREAD

**Issue by leonseng** [volterraedge/terraform-provider-volterra#116]
Version: 0.11.3

Attempted to create `volterra_service_policy`
```
resource "volterra_service_policy" "allow_ns" {
  name       = "allow-ns"
  namespace  = "foo"
  algo       = "FIRST_MATCH"
  any_server = true
  rule_list {
    rules {
      metadata {
        name = "allow-ns"
      }

      spec {
        action = "ALLOW"
        client_selector {
          expressions = [
            "name.ves.io/namespace in (foo)"
          ]
        }
        any_ip           = true
        any_asn          = true
        challenge_action = "DEFAULT_CHALLENGE"
        waf_action {
          none = true
        }
      }
    }
  }
}
```

Received the following error
```
│ Error: Plugin did not respond
│ 
│   with volterra_service_policy.allow_ns,
│   on main.tf line 34, in resource "volterra_service_policy" "allow_ns":
│   34: resource "volterra_service_policy" "allow_ns" {
│ 
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-volterra_v0.11.3 plugin:

panic: interface conversion: interface {} is *schema.Set, not bool

goroutine 235 [running]:
github.com/volterraedge/terraform-provider-volterra/volterra.resourceVolterraServicePolicyCreate(0xc0030e9e30, 0x7c21200, 0xc002bd42c8, 0x2, 0xe1bdec0)
        github.com/volterraedge/terraform-provider-volterra/volterra/resource_volterra_service_policy.go:3506 +0x15ced
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Apply(0xc001aae980, 0xc002bc1b80, 0xc002f52420, 0x7c21200, 0xc002bd42c8, 0x7764601, 0xc00314a6a8, 0xc002f51530)
        github.com/hashicorp/terraform-plugin-sdk@v1.8.0/helper/schema/resource.go:305 +0x375
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Provider).Apply(0xc001aaed80, 0xc002f19a38, 0xc002bc1b80, 0xc002f52420, 0xc002f24868, 0xc002bd4d58, 0x7769da0)
        github.com/hashicorp/terraform-plugin-sdk@v1.8.0/helper/schema/provider.go:294 +0x99
github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc002081850, 0xace6a48, 0xc002beaf30, 0xc002bc8780, 0xc002081850, 0xc002beaf30, 0xc001740ba0)
        github.com/hashicorp/terraform-plugin-sdk@v1.8.0/internal/helper/plugin/grpc_provider.go:885 +0x8a5
github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x7ca8880, 0xc002081850, 0xace6a48, 0xc002beaf30, 0xc002bc8720, 0x0, 0xace6a48, 0xc002beaf30, 0xc000a70000, 0x972)
        github.com/hashicorp/terraform-plugin-sdk@v1.8.0/internal/tfplugin5/tfplugin5.pb.go:3189 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc001556340, 0xad5d038, 0xc0020c6900, 0xc002115200, 0xc00211e8d0, 0xe119020, 0x0, 0x0, 0x0)
        google.golang.org/grpc@v1.29.1/server.go:1082 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc001556340, 0xad5d038, 0xc0020c6900, 0xc002115200, 0x0)
        google.golang.org/grpc@v1.29.1/server.go:1405 +0xccf
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc0004bc200, 0xc001556340, 0xad5d038, 0xc0020c6900, 0xc002115200)
        google.golang.org/grpc@v1.29.1/server.go:746 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.29.1/server.go:744 +0xa5

Error: The terraform-provider-volterra_v0.11.3 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

Error points to this [line](https://github.com/volterraedge/terraform-provider-volterra/blob/main/volterra/resource_volterra_service_policy.go#L3506), referencing a `malicious_user_mitigation_bypass` parameter which I can't seem to find in the [Volterra API](https://www.volterra.io/docs/api/service-policy#operation/ves.io.schema.service_policy.API.Create)

**Comment by brijer** [comment]
I confirm, I have the same problem

**Comment by madhukar32** [comment]
We are working on fixing this bug. ETA is Feb 22

**Comment by madhukar32** [comment]
@leonseng @brijer this is fixed in v0.11.5.
